src/code/padmgr.c

```diff
@@ -94,164 +172,191 @@ void PadMgr_RumbleControl(PadMgr* padMgr) {
 }
 
+/**
+ * Prevents rumble for 3 VI, ~0.05 seconds at 60 VI/sec
+ */
 void PadMgr_RumbleReset(PadMgr* padMgr) {
-    padMgr->rumbleOffFrames = 3;
+    padMgr->rumbleOffTimer = 3;
 }
 
-void PadMgr_RumbleSetSingle(PadMgr* padMgr, u32 ctrlr, u32 rumble) {
-    padMgr->rumbleEnable[ctrlr] = rumble;
-    padMgr->rumbleOnFrames = 240;
+/**
+ * Enables or disables rumble on controller port `port` for 240 VI,
+ * ~4 seconds at 60 VI/sec and ~4.8 seconds at 50 VI/sec
+ */
+void PadMgr_RumbleSetSingle(PadMgr* padMgr, u32 port, u32 rumble) {
+    padMgr->rumbleEnable[port] = rumble;
+    padMgr->rumbleOnTimer = 240;
 }
 
-void PadMgr_RumbleSet(PadMgr* padMgr, u8* ctrlrRumbles) {
+/**
+ * Enables or disables rumble on all controller ports for 240 VI,
+ * ~4 seconds at 60 VI/sec and ~4.8 seconds at 50 VI/sec
+ *
+ * @param enable    Array of 4 u8 containing either true or false to enable or disable rumble for that controller
+ */
+void PadMgr_RumbleSet(PadMgr* padMgr, u8 enable[MAXCONTROLLERS]) {
     s32 i;
 
-    for (i = 0; i < 4; i++) {
-        padMgr->rumbleEnable[i] = ctrlrRumbles[i];
+    for (i = 0; i < MAXCONTROLLERS; i++) {
+        padMgr->rumbleEnable[i] = enable[i];
     }
 
-    padMgr->rumbleOnFrames = 240;
+    padMgr->rumbleOnTimer = 240;
 }
 
-void PadMgr_ProcessInputs(PadMgr* padMgr) {
+/**
+ * Updates `padMgr->inputs` based on the error response of each controller
+ */
+void PadMgr_UpdateInputs(PadMgr* padMgr) {
     s32 i;
     Input* input;
-    OSContPad* padnow1; // original name
+    OSContPad* pad; // original name: "padnow1"
     s32 buttonDiff;
 
     PadMgr_LockPadData(padMgr);
 
-    input = &padMgr->inputs[0];
-    padnow1 = &padMgr->pads[0];
-
-    for (i = 0; i < padMgr->nControllers; i++, input++, padnow1++) {
+    for (input = &padMgr->inputs[0], pad = &padMgr->pads[0], i = 0; i < padMgr->nControllers; i++, input++, pad++) {
         input->prev = input->cur;
 
         if (1) {} // Necessary to match
 
-        switch (padnow1->errno) {
+        switch (pad->errno) {
             case 0:
-                input->cur = *padnow1;
+                // No error, copy inputs
+                input->cur = *pad;
                 if (!padMgr->ctrlrIsConnected[i]) {
                     padMgr->ctrlrIsConnected[i] = true;
                     osSyncPrintf(VT_FGCOL(YELLOW));
                     osSyncPrintf("padmgr: %dコン: %s\n", i + 1, "認識しました"); // "Recognized"
                     osSyncPrintf(VT_RST);
                 }
                 break;
-            case 4:
+            case (CHNL_ERR_OVERRUN >> 4):
+                // Overrun error, reuse previous inputs
                 input->cur = input->prev;
                 LOG_NUM("this->Key_switch[i]", padMgr->ctrlrIsConnected[i], "../padmgr.c", 380);
                 osSyncPrintf(VT_FGCOL(YELLOW));
                 // "Overrun error occurred"
                 osSyncPrintf("padmgr: %dコン: %s\n", i + 1, "オーバーランエラーが発生");
```

Comment:
This patters is so frequent in this file I suspect it's a macro. Otherwise there's not a lot of reason to separate off the error message itself.